### ISSUE TITLE
Can't install realbasicvsr

### ISSUE THREAD

**Issue by iv2985** [adobe-research/SuperGaussian#3]
I can't install realbasicvsr with the instructions provided. I also tried following the directions on BasicVsrPlusPlus, but the required package versions and dependencies are unclear. It worked at one point, but the entire state was never pinned, and now are difficult/impossible for me to replicate.

```
$ python --version
Python 3.8.20
$ pip install torch==1.7.1+cu110 torchvision==0.8.2+cu110 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html
[ ... succeeds ]
$ pip install mmcv-full==1.5.0

Collecting mmcv-full==1.5.0
  Downloading mmcv-full-1.5.0.tar.gz (530 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 530.8/530.8 kB 472.5 kB/s eta 0:00:00
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: -11
  ╰─> [0 lines of output]
      [end of output]


  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

```

**Comment by yshen47** [comment]
Yes, the dependency of realbasicvsr is indeed a bit tricky to sort out. The easiest solution is to use our provided docker image to run and check dependencies. 

Meanwhile, I do recommend considering one stronger video upsampler, upscale-a-video, which is one generative video upsampler that could also integrate into Supergaussian. They recently release codebase, and should expect to achieve better performance than RealBasicVSR.

Hope it helps.

**Comment by MarcWangzhiru** [comment]
This is very interesting and novel work, I want to know how could we make our own datasets that conform to the format? Our own data should be in the same format as the original MVImgNet data. How should we make SuperGS datasets? Thank you very much for your answer.
![image](https://github.com/user-attachments/assets/8e91c31b-0ddf-4161-8691-e123d779a10d)
![image](https://github.com/user-attachments/assets/9cf60bb0-6dc3-478a-8d7c-c1ab1d4b3b08)

**Comment by yshen47** [comment]
Thanks so much for your interest. Short answer is that you need to prepare the format based on nerfstudio format. https://docs.nerf.studio/quickstart/data_conventions.html 

If you check our main.py, we essentially reformat our loaded data in MVImgNet to nerfstudio, and dump the transform.json for the 3dgs to digest.
